### PR TITLE
[zh-cn]: update the translation of Node `normalize()` method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/node/normalize/index.md [hunk 1]
```diff
@@ -1,50 +1,70 @@
...
+title: Node：normalize() 方法
 slug: Web/API/Node/normalize
+l10n:
+  sourceCommit: 312081aabba3885b35a81107b3c2fc53428896c5
 ---
 
 {{APIRef("DOM")}}
 
-`Node.normalize()` 方法将当前节点和它的后代节点”规范化“（normalized）。在一个"规范化"后的 DOM 树中，不存在一个空的文本节点，或者两个相邻的文本节点。
+{{domxref("Node")}} 接口的 **`normalize()`** 方法将指定的节点及其所有子树转化为*规范化*形式。在规范化子树中，子树上的文本节点都不为空的，也没有相邻的文本节点。
```
Comment (T34-active) [inline files/zh-cn/web/api/node/normalize/index.md:10]:
```suggestion
{{domxref("Node")}} 接口的 **`normalize()`** 方法将指定的节点及其所有子树转化为*规范化*形式。在规范化子树中，子树上的文本节点都不为空，也没有相邻的文本节点。
```

File: files/zh-cn/web/api/node/normalize/index.md [hunk 2]
```diff
@@ -1,50 +1,70 @@
...
-wrapper.appendChild(document.createTextNode("Part 1 "));
-wrapper.appendChild(document.createTextNode("Part 2 "));
+wrapper.appendChild(document.createTextNode("第 1 部分"));
+wrapper.appendChild(document.createTextNode("第 2 部分"));
 
-// 这时 (规范化之前),wrapper.childNodes.length === 2
-// wrapper.childNodes[0].textContent === "Part 1 "
-// wrapper.childNodes[1].textContent === "Part 2 "
+let node = wrapper.firstChild;
+let result = "标准化之前：<br/>";
```
Comment (yin1999) [inline files/zh-cn/web/api/node/normalize/index.md]:
```suggestion
let result = "规范化之前：<br/>";
```

File: files/zh-cn/web/api/node/normalize/index.md [hunk 3]
```diff
@@ -1,50 +1,70 @@
...
+title: Node：normalize() 方法
 slug: Web/API/Node/normalize
+l10n:
+  sourceCommit: 312081aabba3885b35a81107b3c2fc53428896c5
 ---
 
 {{APIRef("DOM")}}
 
-`Node.normalize()` 方法将当前节点和它的后代节点”规范化“（normalized）。在一个"规范化"后的 DOM 树中，不存在一个空的文本节点，或者两个相邻的文本节点。
+{{domxref("Node")}} 接口的 **`normalize()`** 方法将指定的节点及其所有子树转化为*规范化*形式。在规范化子树中，子树上的文本节点都不为空，也没有相邻的文本节点。
```
Comment (yin1999) [inline files/zh-cn/web/api/node/normalize/index.md]:
```suggestion
{{domxref("Node")}} 接口的 **`normalize()`** 方法将指定的节点及其所有子树转化为*规范化*形式。在规范化子树中，子树上的文本节点都不为空，且没有相邻的文本节点。
```

**Review by yin1999 — COMMENTED** [review]
额，同一个词的派生词翻译得一致吧（

希望能仔细点翻译

**Review by yin1999 — APPROVED** [review]

**Pull request merged**